src/js/media-chrome-range.js

```diff
@@ -46,7 +46,7 @@ template.innerHTML = `
       box-sizing: border-box;
       display: inline-block;
       position: relative;
-      background: var(--media-control-background, rgba(20,20,30, 0.7));
+      background: var(--media-control-background, var(--media-secondary-color, rgb(20 20 30 / 0.7)));
```

Comment:
We seem to be using `control-background` and `secondary` as interchangeable, and that's feeling unnecessary since they're both _global_.